pkg/metrics/openai_recorder.go

```diff
@@ -71,46 +74,50 @@ func (r *OpenAIRecorder) SetConfigForModel(model string, config *inference.Backe
 		return
 	}
 
+	modelID := r.modelManager.ResolveModelID(model)
+
 	r.m.Lock()
 	defer r.m.Unlock()
 
-	if r.records[model] == nil {
-		r.records[model] = &ModelData{
+	if r.records[modelID] == nil {
+		r.records[modelID] = &ModelData{
 			Records: make([]*RequestResponsePair, 0, 10),
 			Config:  inference.BackendConfiguration{},
 		}
 	}
 
-	r.records[model].Config = *config
+	r.records[modelID].Config = *config
 }
 
 func (r *OpenAIRecorder) RecordRequest(model string, req *http.Request, body []byte) string {
+	modelID := r.modelManager.ResolveModelID(model)
```

Comment:
Does using the model ID in the `OpenAIRecorder` mean that the GUI that displays these requests will also display the SHA256?  Or do we resolve it back to the "friendly" name?  (Or something like `ai/gemma3@sha256:a484b...`?)

Reply:
Currently the UI does not uses/show tags when using a model or inspecting a model. Behind the scenes uses the model ID but the UI does not show that.
With @doringeman we mentioned that we are going to need a `ResolveModelTags(model reference)` to also show all the available tags eventually of a local model.